bots/HACKING.md

```diff
@@ -0,0 +1,38 @@
+# Hacking on the Cockpit Bots
+
+These are automated bots and testing that works on the Cockpit project. This
+includes updating operating system images, bringing in changes from other
+projects, releasing Cockpit and more.
+
+## Environment for the bots
+
+The bots work in containers that are built in the [cockpituous](https://github.com/cockpit-project/cockpituous)
+repository. New dependencies should be added there in the ```tests/Dockerfile```
```

Comment:
I'm not seeing this Dockerfile.

Reply:
It's in the cockpituous project, https://github.com/cockpit-project/cockpituous/blob/master/tests/Dockerfile

Reply:
Yup, that's right.

Reply:
Updated to be more clear.